locales/en.default.schema.json

```diff
@@ -43,8 +43,42 @@
         }
       }
     },
-    "cards": {
-      "name": "Cards",
+    "product_cards": {
+      "name": "Product cards",
+      "settings": {
+        "header__badge": {
+          "content": "Badge"
+        },
```

Comment:
> I think I can remove `header__badge` 🤔 I dont see it being used anywhere but wanted to ensure before

I also just checked and I'm also not seeing `header__badge` used anywhere. We can safely remove all instances from product, collection, and blog cards.